dl/src/main/scala/com/intel/analytics/bigdl/nn/PairwiseDistance.scala

```diff
@@ -55,7 +65,7 @@ class PairwiseDistance[T: ClassTag](
       output.add(diff.pow(ev.fromType[Int](norm)).sum(2))
       output.pow(ev.divide(ev.fromType[Int](1), ev.fromType[Int](norm)))
     } else {
-      require(requirement = false, "input must be vector or matrix")
+      require(requirement = false, inputConstraint)
```

Comment:
A little difference, because the input is table not a tensor, so error information should start with "Each tensor contained in the input" but not ''Each Input".